ansible_kind/kind_setup.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: all
+  strategy: free
+  user: ubuntu
+  tasks:
+   - name: Copy multuclusterkind directory
+     copy: 
+      src: ./scripts/multiclusterkind 
+      dest: /home/ubuntu/ 
+      owner: ubuntu
+      group: ubuntu
+      mode: 0644
+   - name: Prepare VM for kind installation
+     command: bash /home/ubuntu/multiclusterkind/k8s_cluster_setup.sh    
+   - name: Install kind clusters 
+     command: bash /home/ubuntu/multiclusterkind/kind_create_clusters.sh 2
```

Comment:
Only 2 clusters ?

Reply:
We can update it to the number we decide collectively, for testing I set it to 2.